_topic_maps/_topic_map_osd.yml

```diff
@@ -51,45 +77,69 @@ Topics:
   File: gcp-ccs
 ---
 Name: Getting started
-Dir: osd_quickstart
+Dir: osd_getting_started
 Distros: openshift-dedicated
 Topics:
 - Name: Getting started with OpenShift Dedicated
-  File: osd-quickstart
+  File: osd-getting-started
 ---
-Name: Red Hat OpenShift Cluster Manager
-Dir: ocm
+Name: Installing, accessing, and deleting OpenShift Dedicated clusters
+Dir: osd_install_access_delete_cluster
```

Comment:
I noticed that we don't have pages on granting/revoking privileges and access or deleting an OSD cluster in the "Installing, accessing, and deleting OpenShift Dedicated clusters". Maybe we can reuse the modules that are in the OSD "Getting started" page (https://docs.openshift.com/dedicated/osd_quickstart/osd-quickstart.html) in an assembly on that topic for the time being in this section and then build it out with more detail that is provided in the "Getting started" at a later date.

This is something we can look at after the restructure.